collectors/apps.plugin/apps_plugin.c

```diff
@@ -4610,6 +4678,10 @@ static void apps_plugin_function_processes(const char *transaction, char *functi
         add_table_field(wb, "CSysCPU", "Children System CPU Time (100% = 1 core)", false, "bar-with-integer", "bar", "number", 2, "%", CSysCPU_max, "descending", true, false, false, NULL, "sum", true);
         add_table_field(wb, "CGuestCPU", "Children Guest CPU Time (100% = 1 core)", false, "bar-with-integer", "bar", "number", 2, "%", CGuestCPU_max, "descending", true, false, false, NULL, "sum", true);
 
+        // CPU context switches
+        add_table_field(wb, "vCtxSwitch", "Voluntary Context Switches", false, "bar-with-integer", "bar", "number", 2, "switches/s", VoluntaryCtxtSwitches_max, "descending", true, false, false, NULL, "sum", true);
```

Comment:
Why only `switches/s`? The unit of the chart is `context switch`.

Reply:
The unit of both charts is `switches/s`. Where did you find `context switch`?